CHANGELOG.md

```diff
@@ -7,16 +7,23 @@
 > * [CHANGELOG_1.1x.md](./documentation/changelog/CHANGELOG_1.1x.md) - v1.10.0 to v1.19.0
 > * [CHANGELOG_1.2x.md](./documentation/changelog/CHANGELOG_1.2x.md) - v1.20.0 to v1.29.2
 
-# UNRELEASED
+# Node v1.33.0-rc1 / 2025-05-01
+The Lotus v1.33.0-rc1 release introduces experimental v2 APIs with F3 awareness, featuring a new TipSet selection mechanism that significantly enhances how applications interact with the Filecoin blockchain. This release candidate also adds F3-aware Ethereum APIs via the /v2 endpoint while improving existing /v1 APIs and implements intelligent fallback mechanisms between F3 and Expected Consensus.
```

Comment:
Worth also calling out that /v2 APIs are also exposed via gateway.

Reply:
Thanks. Added in: https://github.com/filecoin-project/lotus/pull/13088/commits/575609cfaee50eaeb27d7c3cfdc08aa2f4ad1691